music_assistant/providers/audible/__init__.py

```diff
@@ -111,7 +110,9 @@ async def get_config_entries(
         values[CONF_CODE_VERIFIER] = code_verifier
         values[CONF_SERIAL] = serial
         values[CONF_LOGIN_URL] = login_url
-        webbrowser.open_new_tab(login_url)
+        session_id = str(values["session_id"])
+        mass.signal_event(EventType.AUTH_SESSION, session_id, login_url)
+        await asyncio.sleep(15)
```

Comment:
Reminder for the (core) backlog: create a version of the AuthenticationHelper that uses a form with a textbox to manually paste the redirect url so we dont need to do these wait hacks. For now its fine.